src/App.js

```diff
@@ -0,0 +1,77 @@
+import './App.css';
+import React, { useState, useEffect } from 'react';
+import Sidebar from './components/Sidebar';
+import Editor from './components/Editor';
+import Split from 'react-split';
+import 'react-mde/lib/styles/css/react-mde-all.css';
+import { nanoid } from 'nanoid';
+
+export default function App() {
+  const [notes, setNotes] = useState(() => JSON.parse(localStorage.getItem('notes')) || []);
+  const [currentNoteId, setCurrentNoteId] = useState((notes[0] && notes[0].id) || '');
+
+  useEffect(() => {
+    localStorage.setItem('notes', JSON.stringify(notes));
+  }, [notes]);
+
+  function createNewNote() {
+    const newNote = {
+      id: nanoid(),
+      body: '# This Is a Title',
+    };
+    setNotes(prevNotes => [newNote, ...prevNotes]);
+    setCurrentNoteId(newNote.id);
+  }
+
+  function updateNote(text) {
+    setNotes(oldNotes => {
+      const newArray = [];
+      for (let i = 0; i < oldNotes.length; i++) {
+        const oldNote = oldNotes[i];
+        if (oldNote.id === currentNoteId) {
+          newArray.unshift({ ...oldNote, body: text });
+        } else {
+          newArray.push(oldNote);
+        }
+      }
+      return newArray;
+    });
+  }
+
+  function deleteNote(event, noteId) {
+    event.stopPropagation();
+    setNotes(oldNotes => oldNotes.filter(note => note.id !== noteId));
+  }
+
+  function findCurrentNote() {
+    return (
+      notes.find(note => {
+        return note.id === currentNoteId;
+      }) || notes[0]
+    );
+  }
+
+  return (
+    <main>
+      {notes.length > 0 ? (
+        <Split sizes={[30, 70]} direction='horizontal' className='split'>
+          <Sidebar
+            notes={notes}
+            currentNote={findCurrentNote()}
```

Comment:
Since you're not passing in the arguments in this function, the brackets are not necessarily needed. Check this suggestion.
```suggestion
            currentNote={findCurrentNote}
```